src/microbe_stage/MicrobeColony.cs

```diff
@@ -41,6 +42,17 @@ public Microbe.MicrobeState State
     [JsonProperty]
     public Microbe Master { get; private set; }
 
+    /// <summary>
+    ///   Creates a colony for a microbe, with the given microbe as the master,
+    ///   and handles related updates (like microbe's colony and access to the editor button).
+    /// </summary>
+    /// <remarks> Should be used instead of the colony constructor, unless for loading from Json.</remarks>
```

Comment:
```suggestion
    /// <remarks>Should be used instead of the colony constructor, unless for loading from Json.</remarks>
```
inconsistent spacing here